raml/sfdc_api.raml

```diff
@@ -0,0 +1,52 @@
+# RAML 0.0
+title:
+version: v1
+baseUrl: http://example.com/api
+/alert:
+  post:
+    description: Create or update Case in SFDC
+    responses:
+      200:
+    body:
+      application/json:
+        schema |
+          {"$schema": “http://json-schema.org/schema”,
+            "type":"object"
+            "description": "Alert Data"
+            "properties": {
+              "description":   {"type": "json"},
```

Comment:
because description does not have fixed format - it may contains list of affected hosts or not,  may contains additional fields or not. It depends on alert itself - so if description is json you are able to put here any data you have in alert.